src/QsCompiler/CommandLineTool/Commands/Build.cs

```diff
@@ -79,14 +79,6 @@ public static IEnumerable<Example> UsageExamples
                 HelpText = "Destination folder where the output of the performance assessment will be generated.")]
             public string PerfOutputFolder { get; set; }
 
-            [Option(
```

Comment:
Why was this option removed?

Reply:
I had added the option a while back expecting to use this method as the gating. Instead, I made it so that you specify whether you want debug information in the same way that you specify if you want qir to be emitted & where it should be emitted to (within the assembly properties)